{
  "run_command": "docker run -d --name hacs-postgres -p 5432:5432 -e POSTGRES_DB=hacs -e POSTGRES_USER=hacs -e POSTGRES_PASSWORD=hacs_dev pgvector/pgvector:pg16",
  "run_returncode": 0,
  "run_stdout": "98e2a7afc018ffde869a667221526ff5d3b54f1a1ce8ef4c7cd858a19b45e99a\n",
  "run_stderr": "",
  "logs_stdout": "The files belonging to this database system will be owned by user \"postgres\".\nThis user must also own the server process.\n\nThe database cluster will be initialized with locale \"en_US.utf8\".\nThe default database encoding has accordingly been set to \"UTF8\".\nThe default text search configuration will be set to \"english\".\n\nData page checksums are disabled.\n\nfixing permissions on existing directory /var/lib/postgresql/data ... ok\ncreating subdirectories ... ok\nselecting dynamic shared memory implementation ... posix\nselecting default max_connections ... 100\nselecting default shared_buffers ... 128MB\nselecting default time zone ... Etc/UTC\ncreating configuration files ... ok\nrunning bootstrap script ... ok\nperforming post-bootstrap initialization ... ok\nsyncing data to disk ... ok\n\n\nSuccess. You can now start the database server using:\n\n    pg_ctl -D /var/lib/postgresql/data -l logfile start\n\nwaiting for server to start....2025-08-17 18:54:58.684 UTC [48] LOG:  starting PostgreSQL 16.10 (Debian 16.10-1.pgdg12+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14+deb12u1) 12.2.0, 64-bit\n2025-08-17 18:54:58.685 UTC [48] LOG:  listening on Unix socket \"/var/run/postgresql/.s.PGSQL.5432\"\n2025-08-17 18:54:58.687 UTC [51] LOG:  database system was shut down at 2025-08-17 18:54:58 UTC\n2025-08-17 18:54:58.690 UTC [48] LOG:  database system is ready to accept connections\n done\nserver started\nCREATE DATABASE\n\n\n/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*\n\nwaiting for server to shut down....2025-08-17 18:54:58.873 UTC [48] LOG:  received fast shutdown request\n2025-08-17 18:54:58.874 UTC [48] LOG:  aborting any active transactions\n2025-08-17 18:54:58.875 UTC [48] LOG:  background worker \"logical replication launcher\" (PID 54) exited with exit code 1\n2025-08-17 18:54:58.875 UTC [49] LOG:  shutting down\n2025-08-17 18:54:58.876 UTC [49] LOG:  checkpoint starting: shutdown immediate\n2025-08-17 18:54:58.896 UTC [49] LOG:  checkpoint complete: wrote 922 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.011 s, total=0.021 s; sync files=301, longest=0.004 s, average=0.001 s; distance=4255 kB, estimate=4255 kB; lsn=0/1912070, redo lsn=0/1912070\n2025-08-17 18:54:58.898 UTC [48] LOG:  database system is shut down\n done\nserver stopped\n\nPostgreSQL init process complete; ready for start up.\n\n",
  "logs_stderr": "initdb: warning: enabling \"trust\" authentication for local connections\ninitdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.\n2025-08-17 18:54:58.993 UTC [1] LOG:  starting PostgreSQL 16.10 (Debian 16.10-1.pgdg12+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14+deb12u1) 12.2.0, 64-bit\n2025-08-17 18:54:58.994 UTC [1] LOG:  listening on IPv4 address \"0.0.0.0\", port 5432\n2025-08-17 18:54:58.994 UTC [1] LOG:  listening on IPv6 address \"::\", port 5432\n2025-08-17 18:54:58.996 UTC [1] LOG:  listening on Unix socket \"/var/run/postgresql/.s.PGSQL.5432\"\n2025-08-17 18:54:59.001 UTC [64] LOG:  database system was shut down at 2025-08-17 18:54:58 UTC\n2025-08-17 18:54:59.004 UTC [1] LOG:  database system is ready to accept connections\n",
  "timestamp": "2025-08-17T15:55:01.227745"
}